examples/simple_example.rs

```diff
@@ -0,0 +1,136 @@
+//! Centralized Telescope example with BLS signatures
+
+use alba::centralized_telescope::params::Params;
+use alba::centralized_telescope::proof::Proof;
+use alba::centralized_telescope::CentralizedTelescope;
+use blst::min_sig::{AggregatePublicKey, AggregateSignature, PublicKey, SecretKey, Signature};
+use blst::BLST_ERROR;
+use rand_chacha::ChaCha20Rng;
+use rand_core::{CryptoRng, RngCore, SeedableRng};
+use std::collections::HashMap;
+
+const DATA_LENGTH: usize = 48;
+pub(crate) type Element = [u8; DATA_LENGTH];
+
+struct AlbaSigner {
+    signing_key: SecretKey,
+    verification_key: PublicKey,
+}
+
+struct ThresholdSignature {
+    proof: Proof,
+    key_list: Vec<PublicKey>,
+}
+
+impl AlbaSigner {
+    fn new(rng: &mut (impl RngCore + CryptoRng)) -> Self {
+        let mut ikm = [0u8; 32];
+        rng.fill_bytes(&mut ikm);
+        let sk = SecretKey::key_gen(&ikm, &[])
+            .expect("Error occurs when the length of ikm < 32. This will not happen here.");
+        let pk: PublicKey = sk.sk_to_pk();
+        Self {
+            signing_key: sk,
+            verification_key: pk,
+        }
+    }
+
+    fn sign<const N: usize>(&self, msg: &[u8]) -> [u8; N] {
+        let mut signature_to_byte = [0u8; N];
+        signature_to_byte.copy_from_slice(&self.signing_key.sign(msg, &[], &[]).to_bytes());
+        signature_to_byte
+    }
+}
+
+impl ThresholdSignature {
+    fn aggregate<const N: usize>(
+        alba_signatures: &HashMap<Element, usize>,
```

Comment:
It might be worth adding a comment on the type Element to say that it corresponds here to signatures (so that readers are not confused between elements and alba signatures).

Reply:
I think we should use the signature type directly.